tests/Makefile

```diff
@@ -111,7 +124,7 @@ CHECKING_DIR ?= .
 # Some hack to force the compiler from the test infrastructure with
 # reentrant make:
 ifdef FORCE_CXX
-  CXX=$(FORCE_CXX)
+	CXX=$(FORCE_CXX)
```

Comment:
Why having changed this?

It used to be 2 white spaces.

Reply:
Please use tabulations only where it is really required, which is in the definition of a rule.